kraken-app/src/Pages/Home.jsx

```diff
@@ -1,25 +1,47 @@
 import logo from '../logo.svg';
 import '../App.css';
+import * as React from 'react';
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import CssBaseline from '@mui/material/CssBaseline';
+import MarketplacePreview from '../Components/MarketplacePreview/MarketplacePreview.jsx';
+
 
 function Home() {
   return (
+    <>
     <div className="App-Page">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+
+
+
+
+
+    <CssBaseline />
```

Comment:
Test making the window smaller and see if you can get the entire thing to scale down, the cards stay very large 
![image](https://user-images.githubusercontent.com/60244870/153591150-f264dc20-d238-4c10-b2e5-cc127344006f.png)